screens/ProfileScreen.js

```diff
@@ -1,55 +1,122 @@
 /* profile screen shows user data and has option to edit user data
 also has links to change pw, donate, faq*/
 
-/* replace later with actual display name component -> link to firebase */
-/* replace later with actual email component -> link to firebase */
-/* replace later with actual image component -> link to firebase */
+// TODO: BIANCA
+// fix image cancel bug -> update image url
+// pull email from firebase and display
 
 import React, { useState, useEffect } from "react";
 import { View, StyleSheet, Image, TextInput } from "react-native";
-
+import * as ImagePicker from "expo-image-picker";
+import * as firebase from "firebase";
+import * as Permissions from "expo-permissions";
 import { signOut } from "../utils/auth/auth";
 import { TouchableOpacity } from "react-native-gesture-handler";
 import KitText from "../components/KitText";
 import Colors from "../constants/Colors";
 import ToggleSwitch from "toggle-switch-react-native";
-import { getUsername, updateUsername } from "../utils/db/users";
+import {
+  getUsername,
+  updateUsername,
+  updatePicture,
+  getPicture
+} from "../utils/db/users";
 import { getUserId } from "../utils/auth/auth";
+import uuid from "uuid";
 
 export default function ProfileScreen() {
   const [mode, setMode] = useState("done");
   const [ifOn, setOn] = useState("isOff");
   const [name, setName] = useState("");
   const [email, setEmail] = useState("");
+  const [image, setImage] = useState(null);
+  const [imageUrl, setImageUrl] = useState(null);
   useEffect(() => {
     const currentUser = getUserId();
     console.log("current", currentUser);
     getUsername(currentUser).then(name => {
       setName(name);
     });
   }, []);
+
+  useEffect(() => {
+    Permissions.askAsync(Permissions.CAMERA_ROLL);
+    console.log("current", currentUser);
+    getPicture(currentUser).then(image => {
+      console.log("image", image);
+      setImageUrl(image);
+    });
+  }, []);
+
+  // imageUrl === ""
+  //                 ? require("../assets/images/avatar.png")
+  //                 : { uri: imageUrl }
+
+  if (image) {
```

Comment:
A workaround for the cancel bug could be checking what image is set to if you hit cancel and then checking if image has that value here.